src/core/reactor.cc

```diff
@@ -4042,18 +4061,20 @@ static void sigsegv_action(siginfo_t *info, ucontext_t* uc) noexcept {
     // with addr2line and other tools which expect an
     // address without any added offset (from ASLR or
     // because it's a relocated shared object).
-    print_safe("Segmentation fault: resolved ip: ");
+    print_safe("Segmentation fault: resolved ip: 0x");
```

Comment:
Make it clear these are hex.

Reply:
Nice, but it doesn't fit the "Print incrementally in sigsegv handler" patch title

Reply:
Busted. Would you like me to split it out into a new commit or extend the title/commit description?

Reply:
@xemul I have split this out into a separate PR as requested: https://github.com/scylladb/seastar/pull/2727